data/campaigns/Heir_To_The_Throne/utils/httt_utils.cfg

```diff
@@ -93,17 +128,19 @@
                                 description= _ "This massive blade was created centuries ago by long-forgotten elvish forgemasters, who imbued the bluish steel with an inner magical fire. Tongues of flame dance on the surface, giving the metal a flawless mirrored finish."
                                 cannot_use_message= _ "Only the leader of an army can wield this sword!"
                                 [filter]
-                                    type=Fighter,Commander,Lord,Princess,Battle Princess,Elvish Captain,Elvish Hero,Elvish Marshal,Elvish Champion,Paladin,Elvish Lord,Elvish High Lord
+                                    type=Fighter,Commander,Lord,Princess,Battle Princess,Elvish Captain,Elvish Hero,Elvish Marshal,Elvish Champion,Elvish Lord,Elvish High Lord
```

Comment:
I wonder whether it migth be better to filter on name (id) (would maybe need to be implemented in the c++ engine ), So then it could be some regex check like name=.*blade.*|.*sword.*|.*saber.* this would then also exlude other non-sword melee blade-type attacks like dwarwes axes.